geopandas/tests/test_parquet.py

```diff
@@ -0,0 +1,238 @@
+from __future__ import absolute_import
+import os
+import pytest
+from pandas import DataFrame
+
+from geopandas import GeoDataFrame, GeoSeries, read_file, read_parquet
+from geopandas.datasets import get_path
+from geopandas.io.parquet import _encode_crs, _decode_crs
+from geopandas.testing import assert_geodataframe_equal, assert_geoseries_equal
+
+
+def test_encode_crs():
+    crs = {"init": "EPSG:4326"}
+    assert _encode_crs(crs) == crs
+
+    crs = "+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs"
+    assert _encode_crs(crs) == {"proj4": crs}
+
+    crs = None
+    assert _encode_crs(crs) is None
+
+
+def test_decode_crs():
+    crs = {"init": "EPSG:4326"}
+    assert _decode_crs(crs) == crs
+
+    crs_str = "+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs"
+    crs = {"proj4": crs_str}
+    assert _decode_crs(crs) == crs_str
+
+    crs = None
+    assert _decode_crs(crs) == crs
+
+
+@pytest.mark.parametrize(
+    "test_dataset", ["naturalearth_lowres", "naturalearth_cities", "nybb"]
+)
+def test_to_parquet(test_dataset, tmpdir):
+    """Writing to parquet should not raise errors, and should not alter original
+    GeoDataFrame
+    """
+
+    df = read_file(get_path(test_dataset))
+    orig = df.copy()
+
+    filename = os.path.join(str(tmpdir), "test.pq")
+    df.to_parquet(filename)
+
+    assert os.path.exists(filename)
+
+    # make sure that the original data frame is unaltered
+    assert_geodataframe_equal(df, orig)
+
```

Comment:
maybe you can include the reading here as well (as basic roundtrip test, so combine with test_from_parquet below)